tabular/src/autogluon/tabular/models/abstract/abstract_model.py

```diff
@@ -106,6 +106,10 @@ def __init__(self, path: str, name: str, problem_type: str, eval_metric: Union[s
             self.nondefault_params = list(hyperparameters.keys())[:]  # These are hyperparameters that user has specified.
         self.params_trained = dict()
 
+        #for warm-starting with hyperband scheduler
```

Comment:
Could you document somewhere what these variables are?

Reply:
The variables are updated with comments.